public/docs/ts/latest/cookbook/a11y.jade

```diff
@@ -0,0 +1,444 @@
+
+  We would like to suggest the following rule of thumb when building your applications:
+
+.l-sub-section
+  :marked
+    If there is already a native element inside `HTML` providing the function that you need, use
+    that element instead of writing your own.
+
+:marked
+  So, for example, if you want to accept user text input, use the `input` element instead of contructing something new.
+
+  This way you have to think less about things like focus management, tabindex, etc. and have more time to think about
+  your code.
+
+  But, we know this is not always possible, so this guide will also show you what you can do to make your own
+  components as accessible as possible.
+
+  So without further ado, let us see how easy it is to get big `a11y` wins in our applications!
+
+.l-main-section
+:marked
+  ## Important note on styling in this chapter
+
+.callout.is-important
+  header Angular Pages Do Not Require A Style Library
+  :marked
+    Angular makes no use of the `container`, `row`, `col-xs-12`, `checkbox`, `radio`, `form-group`, `form-control`,
+    and `btn` classes or the styles of any external library. Angular apps can use any CSS library
+    ... or none at all.
+    
+:marked
+  The `container`, `row`, `col-xs-12`, `checkbox`, `radio`, `form-group`, `form-control`, and `btn` classes
+  come from [Twitter Boostrap](http://getbootstrap.com/css/). This is a purely cosmetic addition to prettify
+  the examples so that they are as visually appealing as they are accessible.
+
+  But always remember, just because something looks good, it does not mean that it is accessible.
+
+  The good news is, here we strive to achieve both!
+  
+.l-main-section
+<a id="toc"></a>
+:marked
+  ## Table of contents
+
+  [Accessible form control labels](#form-control-labels)
+
+  [Using keyboard shortcuts](#keyboard-shortcuts)
+
+  [Component that knows its own role](#component-roles)
+
+  [Managing focus](#managin-focus)
+
+.l-sub-section
+  :marked
+    In the example application code when we require unique element id's we will be generating `GUID's`to ensure that they are 
+    unique. You can use your own method to do this, as long as every id on a page is unique. More on this later...
+
+:marked
+  **Feel free to follow along in this [live example](/resources/live-examples/cb-a11y/ts/plnkr.html)**.
+
+.l-main-section
+<a id="form-control-labels"></a>
+:marked
+  ## Accessible form control labels
+
+  Whether you are using already existing `HTML` elements or creating your own rich custom form components, it is crucial to
+  label them. Imagine coming face-to-face with a customer details page on your favourite online store, only to
+  be faced with screen a filled with unlabelled input fields!
+
+  A nightmare right? Users will leave so fast the bouncerate counter will be able to power a small town it will be
+  spinning *THAT* fast.
+
+  We can avoid this from ever happening by simply adding a label for each field. Generaly this is not out problem,
```

Comment:
Typos: "*Generaly* this is not *out* problem," should be "*Generally* this is not *our* problem,"